docs/BindingsOverview.md

```diff
@@ -173,6 +173,9 @@ There are two types of scaling available:
 
 For more information, check documentation on [Runtime Scaling](https://learn.microsoft.com/azure/azure-functions/event-driven-scaling#runtime-scaling). You can configure scaling parameters by going to 'Scale out (App Service plan)' setting on the function app's page. To understand various scale settings, please check the respective sections in [Azure Functions Premium plan](https://learn.microsoft.com/azure/azure-functions/functions-premium-plan?tabs=portal#eliminate-cold-starts)'s documentation.
 
+For information on setup and configuration details, see the [overview](https://learn.microsoft.com/azure/azure-functions/functions-bindings-azure-sql).
```

Comment:
Why are you adding a link to that page here? I don't even see anything about scaling on that page - would https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-azure-sql-trigger?pivots=programming-language-csharp&tabs=isolated-process%2Cpython-v2%2Cportal#enable-runtime-driven-scaling be better to link to instead?

But even that seems to be mostly a duplicate of what we have here already, so I'm not sure what benefit it provides.